ruby_gpg.gemspec

```diff
@@ -49,26 +49,11 @@ Gem::Specification.new do |s|
      "spec/spec_helper.rb"
   ]
 
-  if s.respond_to? :specification_version then
-    current_version = Gem::Specification::CURRENT_SPECIFICATION_VERSION
-    s.specification_version = 3
-
-    if Gem::Version.new(Gem::RubyGemsVersion) >= Gem::Version.new('1.2.0') then
-      s.add_development_dependency(%q<rspec>, [">= 1.2.9"])
-      s.add_development_dependency(%q<yard>, [">= 0"])
-      s.add_development_dependency(%q<cucumber>, [">= 0"])
-      s.add_development_dependency(%q<mocha>, [">= 0.9.8"])
-    else
-      s.add_dependency(%q<rspec>, [">= 1.2.9"])
-      s.add_dependency(%q<yard>, [">= 0"])
-      s.add_dependency(%q<cucumber>, [">= 0"])
-      s.add_dependency(%q<mocha>, [">= 0.9.8"])
-    end
-  else
-    s.add_dependency(%q<rspec>, [">= 1.2.9"])
-    s.add_dependency(%q<yard>, [">= 0"])
-    s.add_dependency(%q<cucumber>, [">= 0"])
-    s.add_dependency(%q<mocha>, [">= 0.9.8"])
-  end
+  s.add_development_dependency(%q<rake>, ["~> 10.0"])
+  s.add_development_dependency(%q<rspec>, ["~> 2.99.0"])
+  s.add_development_dependency(%q<yard>, [">= 0"])
+  s.add_development_dependency(%q<cucumber>, [">= 0"])
+  s.add_development_dependency(%q<mocha>, [">= 1.1.0"])
+  s.add_development_dependency(%q<jeweler>, [">= 2.0.0"])
```

Comment:
@blaix I simplified this block because this seems to target *ancient* environments.